packages/block-editor/src/components/block-list/index.js

```diff
@@ -114,7 +138,9 @@ function BlockList(
 								'is-dropping-horizontally':
 									isDropTarget &&
 									orientation === 'horizontal',
+								'template-part-highlighting': activeTemplatePartId,
```

Comment:
Will this work if `activeTemplatePartId` has ID 0?

Reply:
The ID generally a string, so `"0"` would be truthy.  If it were a numeric value, it would not work.  However this seems a bit different from the standard convention for blockIds, but we could account for that if it seems necessary?

Reply:
Gotcha! I didn't realize it was a string here :)